src/ReplicatedStorage/Client/InstanceComponents/Misc/FallDamage.lua

```diff
@@ -5,30 +5,20 @@ local Trove = require(ReplicatedStorage.Packages.Trove)
 local FallDamage = {}
 FallDamage.__index = FallDamage
 
-function FallDamage:_trackCharacterFall(character: Model, humanoid: Humanoid)
-	local initialHeight = nil
-	local fallHeight = nil
-
-	self._trove:Connect(humanoid.FreeFalling, function(active)
-		if active then
-			initialHeight = character.HumanoidRootPart.Position.Y
-		elseif initialHeight then
-			fallHeight = initialHeight - character.HumanoidRootPart.Position.Y
-			if fallHeight < FallDamageConfig.Threshold then return end
-
-			local damage = (fallHeight - FallDamageConfig.Threshold) * FallDamageConfig.Scale
-			humanoid:TakeDamage(damage)
-		end
-	end)
-end
-
 function FallDamage.new(character: Model)
 	local humanoid = character:WaitForChild("Humanoid")
```

Comment:
Avoid code above declaring self. It makes the function a static/method/constructor mix